tabbed-mode-explainer.md

```diff
@@ -20,22 +20,27 @@ Add a new manifest field `tab_strip` which allows apps to customize the tab stri
 
 ```
 "tab_strip": {
-    "home_tab": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "home_tab": "absent" | {
+        "icons": [...],
+        "scope_patterns": [{"pathname": "..."}]
     },
-    "new_tab_button": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "new_tab_button": "absent" | {
+        "url": <url>,
     },
 },
 ```
 
-The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should never navigate - links clicked from this tab should open in a new app tab. Apps can choose to customize the URL this tab is locked to and the icon displayed on the tab.
+The home tab is a pinned tab that, if enabled for an app, should be present in all app windows. This tab should not navigate outside of the home tab scope (as specified by `scope_patterns`), navigations outside of this scope should open in a new tab. If the `home_tab` field is unset, it will default to `absent`. Apps can choose to customize the icon displayed on the tab.
 
-The new tab button, if present, should open a new tab at a URL that is within the scope of the app. The app may choose to set a custom URL and icon for this button.
+The `home_tab.scope_patterns` field allows the app to set a list of [URLPatterns](https://wicg.github.io/urlpattern/#urlpattern) to define the scope of the home tab. Navigation within the home tab going outside of this scope will be opened in a new tab, and navigation to a URL within the home tab scope will happen in the home tab. The [`URLPattern.baseURL`](https://wicg.github.io/urlpattern/#dom-urlpatterninit-baseurl) will be initialized to the parsed app scope, and apps will only be able to specify the pathname component of URLPattern. If the `scope_patterns` field is unset, then the home tab scope will only include the `start_url`.
 
-If the `tab_strip` field is not present, the particular subfields' `auto` values should be used. The user agent can decide what values to use for `auto`. For example it might set `new_tab_button.url` to be the `start_url`.
+The new tab button, if present, should open a new tab at a URL that is within the scope of the app. The app may choose to set a custom URL to be opened. This URL will default to the `start_url`. If the `new_tab_button` field is unset, it will default to an object with its subfields default values set:
```

Comment:
We should clarify what happens if the new_tab_button url is not in scope (ignored at parse time?).